pyzeebe/grpc/__init__.py

```diff
@@ -0,0 +1,13 @@
+GRPC_CHANNEL_OPTIONS = {
+    # https://docs.camunda.io/docs/product-manuals/zeebe/deployment-guide/operations/setting-up-a-cluster/#keep-alive-intervals
+    # "By default, the official Zeebe clients (Java and Go) send keep alive pings every 45 seconds."
+    "grpc.keepalive_time_ms": 45_000
```

Comment:
Does this fix the bug?

From my (limited) testing I found that I also needed to set `grpc.keepalive_timeout_ms` and `grpc.http2.min_time_between_pings_ms`.

Reply:
Yes, using the repro code in #178, it times out after 1 min 5 sec. `grpc.keepalive_timeout_ms` is by default 20 sec, so that's why it's running longer than 45 sec. 

<details><summary>Logs...</summary>

```
❯ pipenv run python repro.py
2021-07-01 22:16:56,576 DEBUG [123145459896320|Thread-2] zeebe_adapter_base.py:52: Grpc channel connectivity changed to: ChannelConnectivity.IDLE
2021-07-01 22:16:56,576 DEBUG [123145459896320|Thread-2] zeebe_adapter_base.py:56: Connected to localhost:26500
2021-07-01 22:16:56,576 DEBUG [123145459896320|Thread-2] zeebe_adapter_base.py:52: Grpc channel connectivity changed to: ChannelConnectivity.CONNECTING
2021-07-01 22:16:56,576 DEBUG [123145459896320|Thread-2] zeebe_adapter_base.py:62: Connecting to localhost:26500.
2021-07-01 22:16:56,580 DEBUG [123145459896320|Thread-3] zeebe_adapter_base.py:52: Grpc channel connectivity changed to: ChannelConnectivity.READY
2021-07-01 22:16:56,580 DEBUG [123145459896320|Thread-3] zeebe_adapter_base.py:56: Connected to localhost:26500
2021-07-01 22:18:01,593 ERROR [4767251968|MainThread] repro.py:15: Run failed
Traceback (most recent call last):
  File "/Users/krisb/Code/fork/pyzeebe/pyzeebe/grpc_internals/zeebe_workflow_adapter.py", line 27, in create_workflow_instance_with_result
    response = self._gateway_stub.CreateWorkflowInstanceWithResult(
  File "/Users/krisb/.local/share/virtualenvs/pyzeebe-oIjZHgFZ/lib/python3.8/site-packages/grpc/_channel.py", line 923, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/Users/krisb/.local/share/virtualenvs/pyzeebe-oIjZHgFZ/lib/python3.8/site-packages/grpc/_channel.py", line 826, in _end_unary_response_blocking
    raise _InactiveRpcError(state)
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "keepalive watchdog timeout"
	debug_error_string = "{"created":"@1625170681.593180000","description":"Error received from peer ipv6:[::1]:26500","file":"src/core/lib/surface/call.cc","file_line":1063,"grpc_message":"keepalive watchdog timeout","grpc_status":14}"
>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "repro.py", line 9, in <module>
    workflow_instance_key, workflow_result = zeebe_client.run_workflow_with_result(
  File "/Users/krisb/Code/fork/pyzeebe/pyzeebe/client/client.py", line 73, in run_workflow_with_result
    return self.zeebe_adapter.create_workflow_instance_with_result(bpmn_process_id=bpmn_process_id,
  File "/Users/krisb/Code/fork/pyzeebe/pyzeebe/grpc_internals/zeebe_workflow_adapter.py", line 34, in create_workflow_instance_with_result
    self._create_workflow_errors(rpc_error, bpmn_process_id, version, variables)
  File "/Users/krisb/Code/fork/pyzeebe/pyzeebe/grpc_internals/zeebe_workflow_adapter.py", line 46, in _create_workflow_errors
    self._common_zeebe_grpc_errors(rpc_error)
  File "/Users/krisb/Code/fork/pyzeebe/pyzeebe/grpc_internals/zeebe_adapter_base.py", line 90, in _common_zeebe_grpc_errors
    raise ZeebeGatewayUnavailable()
pyzeebe.exceptions.zeebe_exceptions.ZeebeGatewayUnavailable
2021-07-01 22:18:01,595 DEBUG [123145460432896|Thread-4] zeebe_adapter_base.py:52: Grpc channel connectivity changed to: ChannelConnectivity.IDLE
```

</details>